package.json

```diff
@@ -16,6 +16,7 @@
     "react": "16.0.0",
     "react-dom": "16.0.0",
     "tslint": "5.8.0",
+    "tslint-react-native": "0.0.6",
```

Comment:
Should should be a regular dependency. Please run `npm i tslint-react-native -P`